challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>It would be good in each notebook to specify which environment you are expecting this to be run in. I think this is from a conda environment based on the DS CoP pytorch env (?), it would be good to mention that each time, so people can easily pck and run the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I've rewritten my requirements file to encompass all the necessary packages and point to it in each notebook intro. There is only one requirements file since the data engineering packages still work after given in an environment with the restrictions of the machine learning packages